src/components/signupform/signupform.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from "react";
+import { Form, Button } from "react-bootstrap";
+import "./signupform.scss";
+import { MdBorderAll } from "react-icons/md";
+
+class SignUpForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      isLogin: true,
+      email: "",
+      password: "",
+      passwordVerify: "",
+    };
+  }
+
+  handleChange = (params, event) => {
+    event.preventDefault();
+    switch (params) {
+      case "email":
+        this.setState({ email: event.target.value });
+        break;
+      case "password":
+        this.setState({ password: event.target.value });
+        break;
+      case "passwordVerify":
+        this.setState({ passwordVerify: event.target.value });
+        break;
+      case "username":
+        this.setState({ username: event.target.value });
+        break;
```

Comment:
```
const {value, name} = event.target;
this.setState({ [name] : value })
```

This dynamically sets the property value, so we won't need this switch block.

Reply:
Nice approach, we can do that!